README.md

```diff
@@ -116,9 +116,9 @@ Refer to the [docs folder](docs) for design overview and other info on Mobius
 |Run Mobius Shell |<ul><li>[Local](notes/mobius-shell.md#run-shell)</li><li>[YARN](notes/mobius-shell.md#run-shell)</li></ul> | Not supported yet |
 
 ### Useful Links
-* [Configuration parameters in Mobius](/notes/configuration-mobius.md)
-* [Troubleshoot errors in Mobius](/notes/troubleshooting-mobius.md)
-* [Debug Mobius apps](/notes/running-mobius-app.md#debug-mode)
+* [Configuration parameters in Mobius](./notes/configuration-mobius.md)
```

Comment:
The real broken links are those in `notes/CONTRIBUTING.md`, `notes/mobius-shell.md` and the line 62 of `notes/windows-instructions.md`, I add other links of markdown files with prefix `./` to make the link code style stay the same.